tardis/model.py

```diff
@@ -268,6 +268,16 @@ def to_hdf(self, path_or_buf, path='', plasma_properties=None):
         None
 
         """
+        properties = ['t_inner', 'ws', 't_rads', 'v_inner', 'v_outer']
+        for p in properties:
+            output_value = getattr(self, p)
+            if hasattr(output_value, 'si'):
+                output_value = output_value.si.value
+            if np.isscalar(output_value):
+                output_value = [output_value]
+            data = pd.DataFrame(output_value)
```

Comment:
As far as I checked it does not do that automatically.

```python
>>>pd.DataFrame({'a': [1,2], 'b' : [1,2,3]})
ValueError: arrays must all be same length
```

while:
```python
>>> pd.DataFrame({'a': [1,2], 'b' : [1,2]})
   a  b
0  1  1
1  2  2
```

So one would have to take care of manually adding np.nan for the missing values of shorter columns as well.